src/components/SignIn/SignInForm.js

```diff
@@ -87,35 +90,36 @@ export default function SignInForm({ setError }) {
     <Container className={classes.container}>
       <Typography variant="h4" className={classes.welcome}>Welcome back</Typography>
       <FormControl>
-        <RoundedTextField
-          autoFocus
-          variant="outlined"
-          margin="normal"
-          required
-          fullWidth
-          id="email"
-          label="Email"
-          name="email"
-          autoComplete="email"
-          value={email}
-          onChange={({ target: { value } }) => setEmail(value)}
-          disabled={loading}
-        />
-        <RoundedTextField
-          className={classes.textField}
-          variant="outlined"
-          margin="normal"
-          required
-          fullWidth
-          name="password"
-          label="Password"
-          type="password"
-          id="password"
-          autoComplete="current-password"
-          value={password}
-          onChange={({ target: { value } }) => setPassword(value)}
-          disabled={loading}
-        />
+        {[
+          {
+            label: 'Email',
+            type: 'email',
+            autoComplete: 'email',
+            value: email,
+            setState: setEmail,
+            autoFocus: true,
+          },
+          {
+            label: 'Password',
+            type: 'password',
+            autoComplete: 'current-password',
+            value: password,
+            setState: setPassword,
+          },
+        ].map(({
+          label, type, autoComplete, value, setState, autoFocus,
+        }) => (
+          <StyledTextField
+            key={label}
+            label={label}
+            type={type}
+            autoComplete={autoComplete}
+            value={value}
+            onChange={({ target }) => setState(target.value)}
+            autoFocus={autoFocus}
+            disabled={loading}
+          />
+        ))}
```

Comment:
Hmm so it turns out doing array mapping didn't really reduce code reuse much at all and probably made it more confusing, feel free to change it back to how it was before if you want.